posthog/api/github_sdk_versions.py

```diff
@@ -18,13 +21,41 @@
 
 logger = structlog.get_logger(__name__)
 
+# SDK to GitHub repo mapping for lazy-loading individual version dates
+SDK_REPO_MAP = {
+    "web": ("PostHog/posthog-js", "posthog-js@{version}"),
+    "posthog-node": ("PostHog/posthog-js", "posthog-node@{version}"),
+    "posthog-react-native": ("PostHog/posthog-js", "posthog-react-native@{version}"),
+    "posthog-python": ("PostHog/posthog-python", "v{version}"),
+    "posthog-flutter": ("PostHog/posthog-flutter", "{version}"),
+    "posthog-ios": ("PostHog/posthog-ios", "{version}"),
+    "posthog-android": ("PostHog/posthog-android", "android-v{version}"),
+    "posthog-go": ("PostHog/posthog-go", "v{version}"),
+    "posthog-php": ("PostHog/posthog-php", "v{version}"),
+    "posthog-ruby": ("PostHog/posthog-ruby", "v{version}"),
+    "posthog-elixir": ("PostHog/posthog-elixir", "v{version}"),
+    "posthog-dotnet": ("PostHog/posthog-dotnet", "v{version}"),
+}
+
+# Fallback tag formats for SDKs with legacy versions
+# Old posthog-js versions used v-prefixed tags (e.g., "v1.187.2")
+FALLBACK_TAG_TEMPLATES = {
+    "web": "v{version}",  # Legacy format: v-prefixed (e.g., v1.187.2)
+}
+
+# CHANGELOG paths for monorepo SDKs with pre-monorepo history
+CHANGELOG_PATHS = {
+    "posthog-node": "packages/node/CHANGELOG.md",
+    "posthog-react-native": "packages/react-native/CHANGELOG.md",
+}
```

Comment:
Do we really need this @slshults?  
  
Isn't it ok to simply not include/know the changelog/dates for older versions? I'm interested in keeping the code simple here